inc/tickettask.class.php

```diff
@@ -251,6 +251,48 @@ function getRights($interface = 'central') {
       return $values;
    }
 
+   /**
+    * Dropdown of tickettask state
+    *
+    * @param $name   select name
```

Comment:
missing type (syntax is `@param type $varname Documentation`)
also missing `@return`

Reply:
(same for other docblocks)